client/src/components/Applications/AppForm/AppFormSchema.js

```diff
@@ -42,8 +42,10 @@ const FormSchema = (app) => {
           return;
         }
 
+        const fieldName = param.key ?? param.name ?? param.notes?.label;
```

Comment:
This works, though I think you can drop the last bit since I think `param` will always either have `key` or `name`
```suggestion
        const fieldName = param.key ?? param.name;
```